Makefile

```diff
@@ -122,9 +121,18 @@ docker-push: ## Push docker image with the manager.
 
 ##@ Deployment
 
-ifndef ignore-not-found
-  ignore-not-found = false
-endif
+#ifndef ignore-not-found
```

Comment:
This can be removed, rather than commented.

Reply:
removed.